lib/Jsrt/JsrtDebugUtils.cpp

```diff
@@ -0,0 +1,465 @@
+}
+
+void JsrtDebugUtils::AddPropertyToObject(Js::DynamicObject * object, JsrtDebugPropertyId propertyId, Js::Var value, Js::ScriptContext * scriptContext)
+{
+    JsrtDebugUtils::AddVarPropertyToObject(object, propertyId, value, scriptContext);
+}
+
+char16 * JsrtDebugUtils::GetClassName(Js::TypeId typeId)
+{
+    switch (typeId)
+    {
+    case Js::TypeIds_Object:
+    case Js::TypeIds_ArrayIterator:
+    case Js::TypeIds_MapIterator:
+    case Js::TypeIds_SetIterator:
+    case Js::TypeIds_StringIterator:
+                                        return _u("Object");
+
+    case Js::TypeIds_Proxy:             return _u("Proxy");
+    case Js::TypeIds_Array:
+    case Js::TypeIds_NativeIntArray:
+#if ENABLE_COPYONACCESS_ARRAY
+    case Js::TypeIds_CopyOnAccessNativeIntArray:
+#endif
+    case Js::TypeIds_NativeFloatArray:
+    case Js::TypeIds_ES5Array:
+    case Js::TypeIds_CharArray:
+    case Js::TypeIds_BoolArray:
+                                        return _u("Array");
+
+    case Js::TypeIds_Date:
+    case Js::TypeIds_VariantDate:
+                                        return _u("Date");
+
+    case Js::TypeIds_RegEx:             return _u("RegExp");
+    case Js::TypeIds_Error:             return _u("Error");
+    case Js::TypeIds_BooleanObject:     return _u("Boolean");
+    case Js::TypeIds_NumberObject:      return _u("Number");
+    case Js::TypeIds_StringObject:      return _u("String");
+    case Js::TypeIds_Arguments:         return _u("Object");
+    case Js::TypeIds_ArrayBuffer:       return _u("ArrayBuffer");
+    case Js::TypeIds_Int8Array:         return _u("Int8Array");
+    case Js::TypeIds_Uint8Array:        return _u("Uint8Array");
+    case Js::TypeIds_Uint8ClampedArray: return _u("Uint8ClampedArray");
+    case Js::TypeIds_Int16Array:        return _u("Int16Array");
+    case Js::TypeIds_Uint16Array:       return _u("Uint16Array");
+    case Js::TypeIds_Int32Array:        return _u("Int32Array");
+    case Js::TypeIds_Uint32Array:       return _u("Uint32Array");
+    case Js::TypeIds_Float32Array:      return _u("Float32Array");
+    case Js::TypeIds_Float64Array:      return _u("Float64Array");
+    case Js::TypeIds_Int64Array:        return _u("Int64Array");
+    case Js::TypeIds_Uint64Array:       return _u("Uint64Array");
+    case Js::TypeIds_DataView:          return _u("DataView");
+    case Js::TypeIds_Map:               return _u("Map");
+    case Js::TypeIds_Set:               return _u("Set");
+    case Js::TypeIds_WeakMap:           return _u("WeakMap");
+    case Js::TypeIds_WeakSet:           return _u("WeakSet");
+    case Js::TypeIds_SymbolObject:      return _u("Symbol");
+    case Js::TypeIds_Generator:         return _u("Generator");
+    case Js::TypeIds_Promise:           return _u("Promise");
+    case Js::TypeIds_GlobalObject:      return _u("Object");
+    case Js::TypeIds_SpreadArgument:    return _u("Spread");
+
+    default:
```

Comment:
This is not complete set - are we going to add more.
I was looking at lib\Runtime\Types\EdgeJavascriptTypeId.h